index.html

```diff
@@ -316,5 +328,127 @@ <h3 class="about-me__card__title">Skills</h3>
         </div>
       </div>
     </section>
+
+    <!-------CONTACT SECTION ------->
+    <section id="contact">
+      <div class="contact">
+        <div class="contact__container">
+          <h2>
+            I'm always interested in hearing about new projects, so if you'd
+            like to chat please get in touch.
+          </h2>
+
```

Comment:
- [ ] To correctly match the Figma design, kindly ensure that you have used the correct `color_text`, `fonts-weight`,... on the above selected code like the one used on the template.
- [ ] And For best semantics tags, I think it would be better if you use a `<p>` tag instead of a `h2` tag

Reply:
I appreciate your advice, however the styles of both lines of text are the same, and both are part of the h2 tag. So, it's the same than the template. If you look the template in detail, there is an only h2 tag for all that text. So thank you for your advice, but I think it fits with the project’s requirement.

Reply:
@tresorsawasawa could you please be more specific about the color_text and fonts-weight? I have made it as in the template, so sorry but I don't understand you...

Reply:
Hi @sergio-usma 

Here is the used color on Figma `#172b4d` and the you used is `#3a4a42` but for now the `fonts-weght`are the same